tests/users/test_data.py

```diff
@@ -176,17 +176,15 @@ async def test_find_or_create_b2c_user(
     assert user == snapshot(matcher=path_type({"handle": (str,)}))
 
 
-async def test_check_user_does_not_exist(users_data):
+@pytest.mark.parametrize(
+    "no_user_exists, sample_user_data",
+    [(True, None), (False, {"password": "hello_world", "handle": "bill"})],
```

Comment:
* Why do you have to parametrize the user `dict`?
* Shouldn't it be (`True`, `False`) instead of (`True`, `None`)?